src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -103,7 +103,7 @@ const ProfilePicture = ({
 let styles = StyleSheet.create({
   addCircle: {position: 'absolute', top: 25, left: 25},
   profileImage: (size) => {
-    const scale = size <= 25 ? 0.8 : 0.922;
+    const scale = size <= 25 ? 0.8 : 0.91;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the scale factor from `0.922` to `0.91` seems minor and might not have a significant impact on the UI. However, it's important to ensure that this change doesn't affect the layout or positioning of other elements on the screen.
```diff
-    const scale = size <= 25 ? 0.8 : 0.922;
+    const scale = size <= 25 ? 0.8 : 0.91;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->